spec/chrome_integration/gtalk.spec.js

```diff
@@ -0,0 +1,237 @@
+/*
+* GTalk integration tests use 2 test accounts, which should only be used in
+* these tests.  Possible flakiness in these tests may occur:
+* - If multiple users run the tests at the same time, they may send messages
+*   to each other, rather than between the 2 social clients running in the
+*   same test
+* - If tests are run too frequently, GTalk may throttle messages sooner
+*   (return 503 "service unavailable")
+* To prevent flakiness, these tests should only be run by 1 person at a time,
+* with some time (rough estimate 10 minutes) between each attempt.
+ */
+
+var ALICE_EMAIL = 'alicefreedomxmpp@gmail.com';
+var BOB_EMAIL = 'bobfreedomxmpp@gmail.com';
+
+// GTalk gives alicefreedomxpp an anoymized id for bobfreedomxmpp
+// and vice-versa.
+var ALICE_ANONYMIZED_ID = '3bbk93gplwqcn3abf19ivuh6s5@public.talk.google.com';
```

Comment:
Hm. Trying to think if there's anyway to do this that doesn't require the anonymized ID. Is it unsafe to assume that there are only 2 people on their respective buddy lists? (themselves and the other)